src/main/java/com/nike/cerberus/auth/connector/okta/statehandlers/InitialLoginStateHandler.java

```diff
@@ -22,25 +22,6 @@ public InitialLoginStateHandler(AuthenticationClient client, CompletableFuture<A
         super(client, authenticationResponseFuture);
     }
 
-    /**
```

Comment:
Why was this removed, what happens if an OKTA Org does not have MFA turned on, can a user still log in?